aplt/runner.py

```diff
@@ -401,7 +418,7 @@ def parse_ssl_args(args):
         key = os.environ.get("ENDPOINT_SSL_KEY")
         if key and key.startswith(PEM_FILE_HEADER):
             key = StringIO(cert)
-    return cert, key
+    return endpoint, cert, key
```

Comment:
Not a huge deal, but I'm always a bit cautious about adding args to the start or middle of an array.